presto-mixin/dashboards/presto-logs-overview.libsonnet

```diff
@@ -0,0 +1,32 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+local logsDashboard = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main.libsonnet';
+{
+  grafanaDashboards+::
+    if $._config.enableLokiLogs then {
+      local prestoLogs =
+        logsDashboard.new(
+          'Presto logs overview',
+          datasourceName='loki_datasource',
+          datasourceRegex='',
```

Comment:
Missing the datasource regex needed. Currently defaults to `alert-state-history`

Reply:
<img width="1455" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/93c7a12a-b1b1-4c44-baa4-2e014ac284b7">
I was going to add that, but I had noticed that Vitaly commented this on another integration thats in progress.